### ISSUE TITLE
Teal background and Cookie consent at reduced screen widths (Was 264927518 Enhancement Suggestion (Leslie Poulton))

### ISSUE THREAD

**Issue by ghost** [FreeUKGen/FreeUKRegProductIssues#1501]
Issue reported by **Leslie Poulton** at 2023-05-25 06:58:38 UTC
Time: 2023-05-25T06:51:59+00:00
Session ID: 
Problem Page URL: []()
Previous Page URL: [https://www.freereg.org.uk/contacts/new](https://www.freereg.org.uk/contacts/new)
Reported Issue:
Good morning

I am a current transcriber for FreeREG.  I also use this great tool for my own research.  It is great!  As a possible improvement, would you consider using a different background colour to the VIEW in a Search result?  I know that the dark teal colour is part of the logo, but it is very difficult to read all of the details when it is used as a full background colour.  Maybe the paler blue, if you need to use a logo colour.  Just a suggestion.

**Comment by AlOneill** [comment]
Tried to understand the reported issue. I cannot see that teal is being used as the background colour to a search result.

But, I did spot this:

<img width="853" alt="Record_Details_574bad72f493fdf52f010c32___FreeREG_and_A6_Stalogy____June_2023_SETUP____2023_planner_setup_-_YouTube" src="https://github.com/FreeUKGen/FreeUKRegProductIssues/assets/14852554/9210ca07-5330-4a72-8a0e-3569a8a493c1">

In other words, something in the body is not being closed properly or something similar, resulting in the margins being lost. I do not have the energy now to look more deeply, but will do so later if required.

**Comment by AlOneill** [comment]
PS I was on test 3

**Comment by DeniseColbert** [comment]
@SteveBiggs could you go back to Leslie and ask for screenshots of the issue?

**Comment by SteveBiggs** [comment]
I believe this was only an issue on a smart phone and has now been fixed but will check with the OP.

**Comment by Vino-S** [comment]
Sandra to contact Leslie

**Comment by ghost** [comment]
Emailed Leslie this morning, awaiting reply.

**Comment by ghost** [comment]
Leslie has replied there is no change - Leslie noted that it was a laptop she was using and not a phone.

**Comment by SteveBiggs** [comment]
In that case, I think we need a screenshot of the problem as I see a white background. Perhaps also have Leslie tell us what browser and OS she's using?

**Comment by ghost** [comment]
Will email Leslie and see if she can provide the info needed. I have emailed her this evening.

**Comment by ghost** [comment]
![IMG_1903 (1)](https://github.com/FreeUKGen/FreeUKRegProductIssues/assets/20557838/30eeafa8-a010-44c3-85a2-a30874414565)
Leslie finally captures a screenshot of the teal she was talking about.

**Comment by DeniseColbert** [comment]
@CommsCoord to advise Leslie to update her Firfox browser, and clear cookies and cache

@jayto581 to take a look at the teal in the footer 'bleeding' in the CSS

**Comment by jayto581** [comment]
@CommsCoord @DeniseColbert @Vino-S @SteveBiggs 
I was able to reproduce the problem on Firefox, Chrome and Edge. The problem is caused when the browser width window is made too small. 

A temporary fix, is to increase the size of the browser width window to the point when the teal background disappears.

The cause is related to our CSS style sheet implementation for small screens. Further investigation is required to resolve the issue permanently.

**Comment by ghost** [comment]
@jayto581 I think the photo she included showed a split screen. Could this be what is affecting our lady's PC?

**Comment by jayto581** [comment]
@CommsCoord 
No, the split screen is not causing the problem. Anyone who has their Freereg screen too narrow will experience the same problem. A temporary workaround is to make the Freereg screen a little wider and everything should return to normal.

The problem is cause by how Freereg was programmed. The Freereg developers will need to look into it. Thanks for reporting the problem.

**Comment by DeniseColbert** [comment]
@jayto581 identified that from 570px wide and lower, there is code that shouldn't be working in effect. Cookie consent, ad (affecting CEN too) and teal background (just REG). @Vino-S will take a look.

**Comment by Vino-S** [comment]
REady for testing in test3

**Comment by DeniseColbert** [comment]
Steve to test screen width issue

**Comment by SteveBiggs** [comment]
Looks good now.

**Comment by DeniseColbert** [comment]
Closing
